### DISCUSSION TITLE
Multi tenant (client) support

### DISCUSSION THREAD

**Discussion by thefrana** [keystonejs/keystone#7655] [Feature requests]
Hi,
I am trying to build a monorepo that will have many websites, but only one shared CMS. Each website will live in custom database and schema. If I would add a new project I could only register it in the CMS and use it. I think it is better to handle everything from CMS in one project, since it simplifies development of a new features.

Is this planned or somehow possible right now? It should not be that hard. Everything already exists.

Thanks

**Comment by TsarPrince** [comment]
Would love to see that happen!

**Comment by MiguelAngelOne** [comment]
I'm also looking to this feature, but it looks like that no issues are opened for this, do you now if there is a discord or other community channel?

**Reply by unrevised6419** [reply]
https://community.keystonejs.com/
